app/routes/publication-api.js

```diff
@@ -88,4 +88,29 @@ router.post('/entitlementsplans', express.json(), async (req, res) => {
   return res.json(response);
 })
 
+router.post('/readers/:user_id', express.json(), async (req, res) => {
+  const {accessToken} = req.body;
```

Comment:
Should the accessToken be generated server-side instead?

Reply:
Per my other comment, this shouldn't need an access token at all. I'll attempt to remove.